index.html

```diff
@@ -3781,11 +3772,13 @@ <h2>DID Resolution</h2>
             </dt>
             <dd>
 If the resolution is successful, and if the <code>resolve</code> function was
-called, this MUST be a <a>DID document</a> abstract data model (a <a data-cite="INFRA#maps">map</a>) as described in
-<a href="#data-model"></a> that is capable of being transformed
-into a <a>conforming DID Document</a> (representation), using the production
-rules specified by the representation. If the resolution is
-unsuccessful, this value MUST be empty.
+called, this MUST be a <a>DID document</a> abstract data model (a <a
+data-cite="INFRA#maps">map</a>) as described in <a href="#data-model"></a> that
+is capable of being transformed into a <a>conforming DID Document</a>
+(representation), using the production rules specified by the representation.
+The value of <code><a>id</a></code> in the resolved <a>DID document</a> MUST
+match the <a>DID</a> that was resolved. If the resolution is unsuccessful, this
```

Comment:
This looks like a better place for this normative statement. If this section is made non normative during CR phase lets make sure to move this one back out though.